test/cutting/qpd/test_qpd.py

```diff
@@ -148,6 +148,17 @@ def test_decompose_qpd_instructions(self):
                 e_info.value.args[0]
                 == "The number of map IDs (1) must equal the number of decompositions in the circuit (2)."
             )
+        with self.subTest("test_unordered_indices"):
```

Comment:
I don't think this really matters much, but I'll just note that that with the underscores, this doesn't really match the style of the other subtest names.